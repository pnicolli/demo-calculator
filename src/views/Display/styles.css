.display {
    padding: 6px 12px;
    border: 1px solid #EEEEEE;
    background-color: white;
    border-radius: 3px;
    text-align: right;
    font-size: 36px;
}
