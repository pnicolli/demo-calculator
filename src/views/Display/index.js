import React from 'react';
import { number } from 'prop-types';
import './styles.css';

const Display = ({ value }) => (
  <div className="display">
    <span>{value}</span>
  </div>
);

Display.propTypes = {
  value: number,
};

export default Display;
