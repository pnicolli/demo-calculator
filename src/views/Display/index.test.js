import React from 'react';
import { shallow } from 'enzyme';
import Display from './index';

it('matches previous snapshot', () => {
  const tree = shallow(<Display value="42" />);
  expect(tree).toMatchSnapshot();
});
