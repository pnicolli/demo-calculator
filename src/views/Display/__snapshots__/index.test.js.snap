// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`matches previous snapshot 1`] = `
<div
  className="display"
>
  <span>
    42
  </span>
</div>
`;
